changelogs/unreleased/fix-netbox-index-schema-path.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/core
+
+* Fixed missing "path" value of index schema fetched by netbox.
```

Comment:
Issue number, please (in the form `(gh-xxxx)`).

Reply:
Fixed